CHANGELOG.md

```
@@ -1,3 +1,28 @@
+# [6.0.0-next.2](https://github.com/immobiliare/dats/compare/v6.0.0-next.1...v6.0.0-next.2) (2026-05-08)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)